pydm/widgets/label.py

```diff
@@ -31,7 +31,7 @@ def __init__(self, parent=None, init_channel=None):
         QLabel.__init__(self, parent)
         PyDMWidget.__init__(self, init_channel=init_channel)
         self.app = QApplication.instance()
-        self.setTextFormat(Qt.PlainText)
+        self.setTextFormat(Qt.RichText)
```

Comment:
Shouldn't `textFormat` be a designable Qt Property so that we, as the screen designers, can opt-in to rich text if desirable? 

Changing the default settings of a very widely-used widget such as the `PyDMLabel` should be approached with caution, even if the change is seemingly innocuous.